fire/cli/niv/ilæg_revision.py

```diff
@@ -97,100 +92,351 @@ def ilæg_revision(
+                        # Både punktum og komma er accepterede decimalseparatorer
+                        tal = float(r["Ny værdi"].replace(",", "."))
+                    except Exception as ex:
+                        fire.cli.print(
+                            f"    FEJL: {pitnavn} forventer numerisk værdi [{ex}].",
+                            fg="yellow",
+                            bold=True,
+                        )
+                        tal = 0
+                    pi = PunktInformation(infotype=pit, punkt=punkt, tal=tal)
+
+                til_opret.append(pi)
+                punkter_med_oprettelse.add(ident)
+                if alvor:
+                    firedb.indset_punktinformation(opret(sag, punkt.ident, pitnavn), pi)
+                continue
 
             # Ingen ændringer? - så afslutter vi og går til næste element.
-            if pd.isna(r["Sluk"]) and pd.isna(r["Ret tal"]) and pd.isna(r["Ret tekst"]):
+            if r["Sluk"] == r["Ny værdi"] == "":
                 continue
 
             # Herfra håndterer vi kun punktinformationer med indførte ændringer
 
             # Nu kan vi bruge objektid som heltal (ovenfor havde vi brug for NaN-egenskaben)
             oid = int(r["id"])
-
-            # Find det tilsvarende persisterede element
-            try:
-                pinfo = punkt.punktinformationer[infonøgler[oid]]
-            except KeyError:
-                fire.cli.print(
-                    f"    * Ukendt id - ignorerer element '{r}'", fg="red", bold=True
-                )
-                continue
-            anvendelse = pinfo.infotype.anvendelse
-            # print(f"anvendelse={anvendelse}, tekst={r['Ret tekst']}")
-
             if r["Sluk"] == "x":
+                try:
+                    pi = punkt.punktinformationer[infonøgler[oid]]
+                except KeyError:
+                    if alvor:
+                        fire.cli.print(
+                            f"    * Ukendt id - ignorerer element '{oid}'",
+                            fg="red",
+                            bold=True,
+                        )
+                        continue
                 fire.cli.print(f"    Slukker: {pitnavn}")
-                # ...
+                til_sluk.append(pi)
+                punkter_med_slukning.add(punkt.ident)
+                if alvor:
+                    firedb.luk_punktinfo(pi, sluk(sag, punkt.ident, pitnavn))
                 continue
 
-    # Drop sagsevents etc.
+            fire.cli.print(f"    Retter punktinfo-element: {pitnavn}")
+            if pit.anvendelse == PunktInformationTypeAnvendelse.FLAG:
+                pi = PunktInformation(infotype=pit, punkt=punkt)
+            elif pit.anvendelse == PunktInformationTypeAnvendelse.TEKST:
+                pi = PunktInformation(infotype=pit, punkt=punkt, tekst=r["Ny værdi"])
+            else:
+                try:
+                    tal = float(r["Ny værdi"])
+                except Exception as ex:
```

Comment:
```suggestion
                except ValueError as ex:
```
Jeg ser gerne at der gribes en mere specifik exception. Jeg mener `float` smider en `ValueError` hvis den får input den ikke kan konverterer.